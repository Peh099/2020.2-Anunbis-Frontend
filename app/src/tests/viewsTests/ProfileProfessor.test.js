import renderer from 'react-test-renderer';
import ProfileProfessor from '../../views/Profile/Professor';

describe('Snapshot Professor Profile view', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<ProfileProfessor.Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('matches the snapshot', () => {
    const tree = renderer.create(<ProfileProfessor.Body />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
