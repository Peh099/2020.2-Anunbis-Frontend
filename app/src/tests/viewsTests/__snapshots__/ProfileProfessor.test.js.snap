// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Professor Profile view matches the snapshot 1`] = `
<div
  className="sc-bdnxRM gaCZnh"
  height="115px"
  width="240px"
>
  <p>
    Nome Completo: 
  </p>
  <p>
    E-mail: 
  </p>
</div>
`;

exports[`Snapshot Professor Profile view matches the snapshot 2`] = `
<div
  className="sc-bdnxRM jsRjrT"
  height="115px"
  width="430px"
>
  <p>
    Quantidade de avaliações recebidas: 
    0
  </p>
  <p>
    Pontuação média recebida: 
    0
  </p>
</div>
`;
